pkg/resourcecollector/endpoint.go

```diff
@@ -19,6 +19,11 @@ func (r *ResourceCollector) endpointsToBeCollected(
 		return false, fmt.Errorf("error converting to endpoint: %v", err)
 	}
 
+	// Don't migrate the kubernetes endpoint
+	if endpoint.GetName() == "kubernetes" {
```

Comment:
Should we also add the namespace check for "default" namespace. The idea is only to prevent default namespace's "kubernetes" endpoint.

Reply:
Please add release note in the PR.